cli/src/lib/cli.ts

```diff
@@ -27,7 +27,7 @@ export function buildCli(
         yargs.ethscankey = process.env.ETHERSCAN_API_KEY;
       }
       if (!yargs.l1RpcUrl) {
-        yargs.l1RpcUrl = process.env.L1_RPC_CLI;
+        yargs.l1RpcUrl = process.env.L1_RPC_URL;
```

Comment:
We should try to unify envs with webapp (maybe a new lib module?)

Reply:
We can, but I don't believe it's priority. Also, the cli has different behavior for the env variables because they can be set as a cli arg, and that takes priorty.